src/utils/authorization.ts

```diff
@@ -0,0 +1,29 @@
+import { AxiosBasicCredentials } from 'axios'
+
+export const authBasic = (cred: AxiosBasicCredentials) => {
+  if (!cred.username)
+    throw new Error('Username should not be empty or undefined')
```

Comment:
how about empty string?

Reply:
fix and updated

Reply:
this syntax will check undefined and empty string.